inrupt/pod-browser
# Fix/nss no control

This PR fixes bug #SOLIDOS-1062.

`isAcpControlled` is async so we needed to await it before determining if a resource is Wac controlled or not.

To test: 

- Log in with NSS
- Verify the "No control" yellow warning is not displayed and the pod contents are accessible.
